woocommerce/woocommerce
# add: printful placement to add products task

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/woocommerce/issues/48514



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Spin up a WooCommerce Instance, feel free to skip the onboarding profiler as it does not affect the feature
2. Click on the 'Add Products' task on the home screen
3. It should look like the below, and clicking on the "View more product types" should expand the list, keeping printful at the bottom
![image](https://github.com/woocommerce/woocommerce/assets/27843274/4305e5f1-e7cd-478b-8e78-fc583679cd2a)

![image](https://github.com/woocommerce/woocommerce/assets/27843274/90bae91e-b14b-4c6b-bfe8-78a60b623fa1)

4. clicking on the item should open the Printful site in a new window and also record a tracks event under `wcadmin_tasklist_product_printful_advert_click`



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
